site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2147,6 +2147,17 @@ public function getTotalComponentCount($g_id) {
         return intval($this->course_db->row()['cnt']);
     }
 
+    /**
+     * Get an array of Teams for a Gradeable matching the given rotating sections
+     *
+     * @param  string $g_id
+     * @return int
+     */
+    public function getTotalVerifiedComponentCount($g_id) {
+        $this->course_db->query("SELECT count(*) AS cnt FROM gradeable_component_data WHERE GCD_VERIFIER_ID IS NOT NULL", [$g_id]);
+        return intval($this->course_db->row()['cnt']);
+    }
```

Comment:
It would be better to use native PHP type annotations here, since these PHPDoc comments don't provide any information a native annotation wouldn't provide.

```suggestion
    /**
     * Get an array of Teams for a Gradeable matching the given rotating sections
     */
    public function getTotalVerifiedComponentCount(string $g_id): int {
        $this->course_db->query("SELECT count(*) AS cnt FROM gradeable_component_data WHERE GCD_VERIFIER_ID IS NOT NULL", [$g_id]);
        return intval($this->course_db->row()['cnt']);
    }
```